lib/src/authorization_code_grant.dart

```diff
@@ -59,6 +59,10 @@ class AuthorizationCodeGrant {
   /// documentation.
   final Uri authorizationEndpoint;
 
+  /// A function used to parse parameters out of responses from hosts that do not
+  /// respond with application/json or application/x-www-form-urlencoded bodies.
+  final Function getParameters;
```

Comment:
I don't think this should be a public field. Also, it should use a (private) typedef—the type `Function` effectively doesn't mean anything to the type system.